data/coalition/coalition missions.txt

```diff
@@ -3171,3 +3171,539 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	"Oh, my friend Anna Fangvin is publishing a book here. I brought her here."`
+					goto "kst-1 anna"
+			label "kst-1 not only alien"
+			`	"If more of you, here there are, then who are they?"`
+			choice
+				`	"It's just two; my friend Anna Fangvin and I. She's publishing a book in your space. I brought her here."`
+					goto "kst-1 anna"
+			label "kst-1 endonym"
+			`	"'Humans', you are? Know that, I never did. For correcting me and educating me on your fellow kind, I am thankful," says the Kimek. "So, the other human, who is she?"`
+			choice
+				`	"Oh, that's my friend Anna Fangvin. She's here to publish her book in Coalition space."`
+			label "kst-1 anna"
+			`	"Anna Fangvin, her name is? An interesting name, it is, and her book, just as interesting it will be when it is released. But know the title of the book, I do not, so if tell me you will, appreciate it I will."`
+			choice
+				`	"It is called 'Beyond the Hyperdrive'. It could very much be years before release, but I'm glad you're taking interest."`
+				`	"It is called 'Beyond the Hyperdrive'.`
+			`	"So, 'Beyond the Hyperdrive', it is, you say? Thankful, I am, and on its release, buy the book I will remember to."`
+			`	The Kimek walks away after the two of you exchange farewells, leaving you to your own devices.`
+				goto "anna approaches"
+			label "kst-1 ignore"
+			`	"Apologies for invading your personal space, I give." The Kimek quickly walks away.`
+			label "anna approaches"
+			`	After a few hours, you see Anna enter the lounge and walk toward you. She takes a seat next to you.`
+			choice
+				`	"How did the interview go?"`
+			`	"They told me that due to the censorship and review process, I'll have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please remember to come back and meet me."`
+			choice
+				`	"Why don't you just go back to human space?"`
+					goto stay
+				`	"I'll make sure to do that."`
+			label thanks
+			`	"Thanks again, <first> <last>. I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see."`
+			choice
+				`	"Just helping out."`
+				`	"I'm happy to have helped you."`
+			`	"Again, thank you. You will not be needed anymore for the process."`
+			choice
+				`	(Leave.)`
+			`	You return to the spaceport the way you came and return to your ship. You are slightly anxious about the outcome of the process and whether Anna's book will even make it through the process and be published, but you have more important things to attend to for the next six months.`
+				decline
+			label stay
+			`	"I need to stay here so I can answer their questions during the review process. Don't worry about me."`
```

Comment:
She can just do that? Does she not have any sort of connections or obligations in human space? What about her friends and family?